### PR TITLE
Avoid throwing exception when trying to determine versions

### PR DESCRIPTION
**Pull request by birgirst**
This can happen if an operator would specify an invalid baseurl where
release metadata can be found.

### CODE REVIEW

File: fiaas_skipper/web/frontend.py [hunk 1]
```diff
@@ -39,9 +43,14 @@ def index():
...
     versions = {}
     for tag in ("stable", "latest"):
-        channel = frontend.release_channel_factory("fiaas-deploy-daemon", tag)
-        image = channel.metadata["image"]
-        versions[tag] = image.split(":")[-1]
+        try:
+            channel = frontend.release_channel_factory("fiaas-deploy-daemon", tag)
+            image = channel.metadata["image"]
+            version = image.split(":")[-1]
+        except requests.exceptions.ConnectionError:
```
Comment (oyvindio) [inline fiaas_skipper/web/frontend.py]: This patch works fine as it is, but I think that the problem could be expressed more clearly by for example catching ConnectionError and logging an error that is useful for troubleshooting the underlying problem (could not connect/bad url) in ReleaseChannel.__call__ and then raise a custom exception with a name that describes the problem that callers (i.e. this method) have to handle (latest version for tag is not available). What do you think?

**Comment by mortenlj** [comment]
The invalid syntax error is because Prospector is run using Python 2. I have merged a configuration for Codacy to master that should fix this, but since Codacy is super slow these days it will take some time to confirm. I think we should just ignore it and move on.

**Comment by birgirst** [comment]
Feel free to merge this @mortenlj

**Comment by mortenlj** [comment]
> Feel free to merge this @mortenlj

Yup, just waiting for the builds to complete. :smiley:

**Pull request merged**